src/VisualStudio/Core/Def/Implementation/TaskList/ExternalErrorDiagnosticUpdateSource.cs

```diff
@@ -168,13 +168,13 @@ public void ClearErrors(ProjectId projectId)
 
             // Update the state to clear diagnostics and raise corresponding diagnostic updated events
             // on a serialized task queue.
-            _taskQueue.ScheduleTask(nameof(ClearErrors), async () =>
+            _taskQueue.ScheduleTask(nameof(ClearErrors), () =>
```

Comment:
aside: i think we should get rid of this taskqueue.  I have a feeling it is the source of lots of potential bugs around having a consistent model of all diagnostics.  esp. as these are all just sync operations, i really dont' get why we dont' just apply them synchronously.

Reply:
I believe the task queue was introduced to minimize the work done in these callbacks, which is likely to slow down the build. Won't all these code go away from Roslyn once we move to LSP diagnostics?

Reply:
taht would be the hope :)  we certainly shouldn't have an ExternalErrorDiagUpdateSource in roslyn then :)